product_profile_example/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product Profile Example
+=======================
+
+This module is a use case of **Product Profile Example** module.
+
+See its description.
+
+
+
+Configuration
+=============
+
+Optionnaly:
+
+* Settings > Configuration > General Settings: check 'Display Profile fields' and apply
+  (only if you want display hidden fields)
+
+
+
+Usage
+=====
+
+Create or modify a product template or a product variant. Assign a value to the profile field.
+Then, all fields which depend on this profile will be set to the right value at once.
+
+Bug Tracker
+===========
+
+See 'Bug tracker' in the README.rst of Product Profile module
```

Comment:
It's a demo module with data. I think it's better to gather issues on the main module because root cause comes from product_profile